koku/masu/database/trino_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -190,9 +190,9 @@ SELECT cast(uuid() as varchar) as uuid,
             THEN  split_part(unitofmeasure, ' ', 2)
         ELSE unitofmeasure
     END) as unit_of_measure,
-    sum(coalesce(nullif(azure.quantity, 0), azure.usagequantity)) as usage_quantity,
+    sum(coalesce(nullif(azure.quantity, 0), 0)) as quantity,
     coalesce(nullif(azure.billingcurrencycode, ''), nullif(azure.currency, ''), azure.billingcurrency) as currency,
-    sum(coalesce(nullif(azure.costinbillingcurrency, 0), azure.pretaxcost)) as pretax_cost,
+    sum(coalesce(nullif(azure.costinbillingcurrency, 0), 0)) as costinbillingcurrency,
```

Comment:
We should also stick to `pretax_cost`. `costinbillingcurrency` is not a field in our tables.

```suggestion
    sum(azure.costinbillingcurrency) as pretax_cost,
```